docs/termux.md

```diff
@@ -104,7 +103,7 @@ you will get node version `v16.15.0`
 5. Now install code-server
 
 ```sh
-yarn global add code-server
+npm install --global code-server
 ```
```

Comment:
Should this simply be a link to https://coder.com/docs/code-server/latest/npm - rather than re-explaining the process?

Reply:
That makes sense to me. Either works - I'll leave that up to you!

Reply:
I like consistency and less duplication... Latest amended commit just redirects to existing guide.